src/libsystemd/sd-device/sd-device.c

```diff
@@ -1429,6 +1429,50 @@ _public_ int sd_device_get_seqnum(sd_device *device, uint64_t *ret) {
         return 0;
 }
 
+int device_get_diskseq(sd_device *device, int fd, bool verify, uint64_t *ret) {
```

Comment:
i don't understand this function. this does considerably more than just getting the diskseq. It also verifies, and sometimes returns zero?